lib/raidenclient/RaidenClient.ts

```diff
@@ -272,14 +278,17 @@ class RaidenClient extends SwapClient {
   }
 
   /**
-   * Returns the total balance available across all channels.
+   * Returns the total balance available across all channels for a specified currency.
    */
-  public channelBalance = async (): Promise<ChannelBalance> => {
-    // TODO: refine logic to determine balance per token rather than all combined
-    const channels = await this.getChannels();
+  public channelBalance = async (currency?: string): Promise<ChannelBalance> => {
```

Comment:
Let's add a test for this one to:
https://github.com/ExchangeUnion/xud/blob/master/test/jest/RaidenClient.spec.ts

Reply:
Done.